static/app/components/profiling/profilingTransactionHovercard.tsx

```diff
@@ -0,0 +1,301 @@
+import {CSSProperties, Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {Hovercard} from 'sentry/components/hovercard';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {Organization, Project} from 'sentry/types';
+import {DURATION_UNITS} from 'sentry/utils/discover/fieldRenderers';
+import {getShortEventId} from 'sentry/utils/events';
+import {useFunctions} from 'sentry/utils/profiling/hooks/useFunctions';
+import {useProfileEvents} from 'sentry/utils/profiling/hooks/useProfileEvents';
+import {
+  generateProfileFlamechartRouteWithQuery,
+  generateProfileSummaryRouteWithQuery,
+} from 'sentry/utils/profiling/routes';
+import {useLocation} from 'sentry/utils/useLocation';
+import usePageFilters from 'sentry/utils/usePageFilters';
+import {ContextTitle} from 'sentry/views/eventsV2/table/quickContext/styles';
+import {getProfilesTableFields} from 'sentry/views/profiling/profileSummary/content';
+
+import Button from '../button';
+import Link from '../links/link';
+import LoadingIndicator from '../loadingIndicator';
+import PerformanceDuration from '../performanceDuration';
+
+interface ProfilingTransactionHovercardProps {
+  organization: Organization;
+  project: Project;
+  transaction: string;
+}
+
+export function ProfilingTransactionHovercard(props: ProfilingTransactionHovercardProps) {
+  const {project, transaction, organization} = props;
+  const {query} = useLocation();
+
+  const linkToSummary = generateProfileSummaryRouteWithQuery({
+    query,
+    orgSlug: organization.slug,
+    projectSlug: project.slug,
+    transaction,
+  });
+
+  const triggerLink = <Link to={linkToSummary}>{transaction}</Link>;
+
+  if (!organization.features.includes('profiling-dashboard-redesign')) {
+    return triggerLink;
+  }
```

Comment:
i do think its time for a `useFeature` hooks, i have a draft but its a change that requires more thought https://github.com/getsentry/sentry/pull/42951